binder/views.py

```diff
@@ -1161,10 +1248,9 @@ def _parse_order_by(self, queryset, field, request, partial=''):
 		return (queryset, partial + head, nulls_last)
 
 
-
-	def search(self, queryset, search, request):
+	def _search_base(self, search, request):
 		if not search:
-			return queryset
+			return ~Q(pk__in=[])
```

Comment:
Lets add a comment here why we do it like this, otherwise in the future people will be like hey let me do it the logical way, going back to the older and slower version.